master/src/main/resources/application-local.yaml

```
@@ -47,4 +47,4 @@ management:
   endpoints:
     web:
       exposure:
-        include: [ "health" ]
+        include: [ "health" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)